connector/openshift/openshift.go

```diff
@@ -28,30 +28,32 @@ const (
 
 // Config holds configuration options for OpenShift login
 type Config struct {
-	Issuer       string   `json:"issuer"`
-	ClientID     string   `json:"clientID"`
-	ClientSecret string   `json:"clientSecret"`
-	RedirectURI  string   `json:"redirectURI"`
-	Groups       []string `json:"groups"`
-	InsecureCA   bool     `json:"insecureCA"`
-	RootCA       string   `json:"rootCA"`
+	Issuer            string   `json:"issuer"`
+	ClientID          string   `json:"clientID"`
+	ClientSecret      string   `json:"clientSecret"`
+	RedirectURI       string   `json:"redirectURI"`
+	Groups            []string `json:"groups"`
+	InsecureCA        bool     `json:"insecureCA"`
+	RootCA            string   `json:"rootCA"`
+	IncludeHostRootCA bool     `json:"includeHostRootCA"`
```

Comment:
Why do you call this "host"? Apart from the fact that host is typically seen as opposed to a container (which would make it technically inaccurate, as the root CAs are loaded from the container, not the host), it also conflicts with the remaining nomenclature in Go (`SystemCertPool`).

Also, the plural form should be used here - `IncludeSystemCAs` (note that `RootCA` should actually be called `RootCAs`, because it can contain multiple certs, but there's no renaming it now. However, let's do it correctly for anything new we introduce)

Reply:
> Why do you call this "host"?

Looking at the doc for some connectors (i.e. [LDAP](https://dexidp.io/docs/connectors/ldap/#configuration)) it is mentioned within there, as dex is _not_ strictly concerned about running in a containerized environment, so from their point of view the `host` usage is pretty fine IMO.
But I don't have too strong of an opinion to keep it, so renamed it to `IncludeSystemCAs`.